src/Artsy/Router/Boot.tsx

```diff
@@ -19,74 +19,69 @@ import {
-// issue will be fixed
-const { GlobalStyles } = injectGlobalStyles<{
-  suppressMultiMountWarning: boolean
-}>()
+const { GlobalStyles } = injectGlobalStyles()
 
-@track(null, {
-  dispatch: data => {
-    Events.postEvent(data)
-  },
-})
-export class Boot extends React.Component<BootProps> {
-  componentDidMount() {
-    const env = sd.NODE_ENV || (process.env && process.env.NODE_ENV)
-    if (env === "production") {
+export const Boot = track(null, {
+  dispatch: Events.postEvent,
+})((props: BootProps) => {
+  /**
+   * Let our end-to-end tests know that the app is hydrated and ready to go; and
+   * if in prod, initialize Sentry.
+   */
+  useEffect(() => {
+    document.body.setAttribute("data-test", "AppReady") //
+
+    if (getENV("NODE_ENV") === "production") {
       Sentry.init({ dsn: sd.SENTRY_PUBLIC_DSN })
     }
+  }, [])
 
-    // Let our end-to-end tests know that the app is hydrated and ready to go
-    document.body.setAttribute("data-test", "AppReady")
-  }
-
-  render() {
-    const { children, context, headTags = [], ...props } = this.props
-    const contextProps = {
-      ...props,
-      ...context,
-    }
+  const {
+    children,
+    context,
+    headTags = [],
+    onlyMatchMediaQueries,
+    ...rest
+  } = props
 
-    return (
-      <Theme>
-        <HeadProvider headTags={headTags}>
-          <StateProvider>
-            <Artsy.SystemContextProvider {...contextProps}>
-              <ErrorBoundary>
-                <MediaContextProvider onlyMatch={props.onlyMatchMediaQueries}>
-                  <ResponsiveProvider
-                    mediaQueries={themeProps.mediaQueries}
-                    initialMatchingMediaQueries={
-                      props.onlyMatchMediaQueries as any
-                    }
-                  >
-                    <Grid fluid>
-                      <GlobalStyles suppressMultiMountWarning />
-                      {children}
-                      {process.env.NODE_ENV === "development" && (
-                        <BreakpointVisualizer />
-                      )}
-                    </Grid>
-                  </ResponsiveProvider>
-                </MediaContextProvider>
-              </ErrorBoundary>
-            </Artsy.SystemContextProvider>
-          </StateProvider>
-        </HeadProvider>
-      </Theme>
-    )
+  const contextProps = {
+    ...rest,
+    ...context,
   }
-}
 
-// Tests
-GlobalStyles.displayName = "GlobalStyles"
-Grid.displayName = "Grid"
+  return (
+    <Theme>
+      <HeadProvider headTags={headTags}>
+        <StateProvider>
+          <SystemContextProvider {...contextProps}>
+            <ErrorBoundary>
+              <MediaContextProvider onlyMatch={onlyMatchMediaQueries}>
+                <ResponsiveProvider
+                  mediaQueries={themeProps.mediaQueries}
+                  initialMatchingMediaQueries={onlyMatchMediaQueries as any}
+                >
+                  <Grid fluid>
```

Comment:
The grid component (in palette) has a max-width set on it.

Reply:
Ahh -- so we had max-width set in a few areas. I can update palette with a new maxWidth prop and pass in systemContext maxwidth here. Will do a quick cycle over there and then rebase this PR.